client/submitAssignment.js

```diff
@@ -1,37 +1,58 @@
+let validation = false;
 
-// Creates a http POST request to the server and sends assignment tar ball and cmg args 
-$("#mainForm").submit(function(event) {
-    var formData = new FormData(this);
-    event.stopPropagation();
-    event.preventDefault();
+function uploadForm() {
+    validateForm();
+    if(validation) {
+        let formData = new FormData(document.getElementById("mainForm"));
+        $.ajax({
+            url: "http://localhost:8082/upload",
```

Comment:
this will fail if the http server is running on a port other than 8082.

Reply:
Changed it. Added a `config.js` file to hold the server port number.